templates/nextflow-ecs-task-definition.yaml

```diff
@@ -159,12 +224,14 @@ Resources:
           DependsOn:
             - ContainerName: !Ref RedisContainerName
               Condition: START
+            - ContainerName: !Sub '${CronContainerName}-MigrateDb'
+              Condition: SUCCESS
```

Comment:
Based on the [documentation](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDependency.html) - I think this does mean that the cron container image will only start after the migrateDb container has successfully run with a 0 exit status.

Reply:
That's my interpretation as well.